include/entity.h

```diff
@@ -190,8 +190,9 @@ typedef struct PACKED {
     /* 0x8A */ s16 unk8A;
     /* 0x8C */ struct Entity* parent;
     /* 0x90 */ s32 unk90;
-    /* 0x94 */ s32 unk94;
-    /* 0x98 */ s32 unk98;
+    /* 0x94 */ struct Entity* other;
```

Comment:
Pointers need special consideration since they are 8 bytes on 64-bit platforms, you're hitting asserts due to this https://github.com/Xeeynamo/sotn-decomp/blob/247f58f90ef50ab76f68f7eea7a546cacd549f50/include/entity.h#L1549

Reply:
Your ET_Weapon is:
```
typedef struct PACKED {
    /* 0x7C */ s16 lifetime;
    /* 0x7E */ s16 unk7E;
    /* 0x80 */ s16 unk80; // Y?
    /* 0x82 */ s16 unk82; // X?
    /* 0x84 */ struct Entity* some_ent;
    /* 0x88 */ s16 childPalette;
    /* 0x8A */ s16 unk8A;
    /* 0x8C */ struct Entity* parent;
    /* 0x90 */ s32 unk90;
    /* 0x94 */ struct Entity* other;
    /* 0x98 */ s16 unk98;
    /* 0x9A */ s16 unk9A;
    /* 0x9C */ s32 accelerationX;
    /* 0xA0 */ s32 accelerationY;
    /* 0xA4 */ s32 unkA4;
    /* 0xA8 */ s32 unkA8;
    /* 0xAC */ u8 anim;
    /* 0xAD */ u8 unkAD;
    /* 0xAE */ s16 equipId;
} ET_Weapon;
```

` struct Entity* other;` is new. It can be 4 bytes on 32-bit platforms and 8 bytes on 64-bit platforms. 

Your ET_Player is:
```
typedef struct PACKED {
    Primitive* prim;
    char pad[0x24];
    s16 unkA4;
    s16 unkA6;
    void* unkA8;
    u8 anim;
    char pad2[0x8];
    struct Entity* unkB8;
} ET_Player;
```

There's no corresponding padding in ET_Player, so if `other` ends up being 8 bytes, anim will be misaligned